### PR TITLE
Feature/ascii test

### LINKED ISSUE #70: Asciify Testing
test asciify function in jest

### PR DESCRIPTION
**Pull request by elizasohn**

### CODE REVIEW

File: artscii/src/tests/Asciify.test.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+import convertToGrayScales from '../services/convertToGrayScales';
+import dummyImgSrc from './dummyImgSrc';
+import drawAscii from '../services/drawAscii';
+
+const image = new Image();
+image.src = dummyImgSrc;
+const width = 256;
+const height = 128;
+const canvas = document.createElement('canvas');
+const context = canvas.getContext('2d');
```
Comment (srimel) [inline artscii/src/tests/Asciify.test.js:11]: Unit testing looks great and seems to have good coverage with all testing passed. However, I had to manually `npm install canvas` to get past an exception being thrown here in your test suite. Could you also include the dependency change in the package.json for this PR?

**Review by srimel — CHANGES_REQUESTED** [review]

File: artscii/src/tests/Asciify.test.js (hunk 1 above)
Comment (unfinishedideas) [reply artscii/src/tests/Asciify.test.js:11]: Huh, yeah that's strange. I had the same issue. When line 10 gets called (const canvas = document.createElement('canvas');) it just sets canvas to an undefined. Looks like a good suite of tests though!

Comment (anamika8) [reply artscii/src/tests/Asciify.test.js:11]: I got the same issue but adding that dependency will fix it. It's a minor one. It looks like a good coverage of unit tests has been implemented.

Comment (elizasohn) [reply artscii/src/tests/Asciify.test.js:11]: Woops, sorry about that - added the dependency!

**Review by unfinishedideas — APPROVED** [review]
Sweet, seems to be fixed; approved!

**Review by srimel — APPROVED** [review]
Thanks!

**Review by anamika8 — APPROVED** [review]
Looks great to me!

**Pull request merged**